### PR TITLE
Update CRD spec with API key auth option

### PR DESCRIPTION
**Pull request by guicassolato**
Missing in #55

### CODE REVIEW

File: config/crd/bases/config.authorino.3scale.net_services.yaml [hunk 1]
```diff
@@ -73,6 +73,21 @@ spec:
...
                 properties:
+                  api_key:
+                    properties:
+                      key_selector:
+                        type: string
+                      label_selectors:
+                        additionalProperties:
+                          type: string
+                        type: object
+                      location:
```
Comment (didierofrivia) [inline config/crd/bases/config.authorino.3scale.net_services.yaml:84]:
```suggestion
                      in:
```

File: config/crd/bases/config.authorino.3scale.net_services.yaml [hunk 2]
```diff
@@ -73,6 +73,21 @@ spec:
...
+                        type: string
+                    required:
+                    - key_selector
+                    - label_selectors
+                    - location
```
Comment (didierofrivia) [inline config/crd/bases/config.authorino.3scale.net_services.yaml:89]:
```suggestion
                    - in
```

File: config/crd/bases/config.authorino.3scale.net_services.yaml (hunk 1 above)
Comment (guicassolato) [reply config/crd/bases/config.authorino.3scale.net_services.yaml:84]: This is auto-generated

**Comment by guicassolato** [comment]
Closing in favour of #66

**Pull request closed without merging**